twine-cert.txt

```diff
@@ -1322,6 +1322,30 @@
     de = Leider ist eine automatische Prüfung nicht möglich. Überprüfen Sie die unten markierten Werte bitte selbst.
     en = Sorry, an automatic check is not possible for this certificate. Please review the information marked out below yourself.
 
+[check_validity_no_rules_status]
+    de = Keine Regeln vorhanden
+    en = No rules available
+
+[check_validity_no_rules_title]
+    de = Keine Einreiseregeln vorhanden
+    en = No entry rules available
+    
+[check_validity_no_rules_copy]
+    de = Leider ist eine automatische Prüfung nicht möglich, da für dieses Land keine Einreiseregeln hinterlegt sind. Bitte informieren Sie sich auf der Internetseite des Auswärtigen Amts.
```

Comment:
It would make sense to actually add a link to the website https://www.auswaertiges-amt.de here.

Reply:
Danke Tim für dein Feedback! Eine Verlinkung hatten wir bereits angedacht, jedoch ist ein zielführender Link auf die Covid-19 Hinweise nicht für unsere English-sprachigen User verfügbar. Da es sich um eine Behörde des Bundes handelt, sind wir nicht befugt dort aktiv zu Content zu verlinken und sollen das deshalb aktuell nicht machen. Danke für dein Verständnis.

Reply:
@Staeffy Danke für das Feedback! Schade, aber verständlich!